ntc_templates/templates/alcatel_aos_show_interfaces_port.textfsm

```diff
@@ -0,0 +1,16 @@
+Value PORT ((\d+\/?)+)
+Value WTR_RUNNING (\#?)
+Value PERMANENT_SHUTDOWN (\*?)
+Value ADMIN_STATUS (\S+)
+Value LINK_STATUS (\S+)
+Value VIOLATIONS (\S+)
+Value RECOVERY_TIME (\d+)
+Value RECOVERY_MAX (\d+)
+Value WTR_TIME (\d+)
+Value ALIAS (.*)
+
+Start
+  ^\s*-+\+
+  ^\s*${WTR_RUNNING}${PERMANENT_SHUTDOWN}${PORT}\s+${ADMIN_STATUS}\s+${LINK_STATUS}\s+${VIOLATIONS}\s+${RECOVERY_TIME}\s+${RECOVERY_MAX}\s+${WTR_TIME}\s+"${ALIAS}"\s*$$ -> Record
+  ^\s*
```

Comment:
```suggestion
  ^\s*$$
```
We don't want to just catch any line that stars with a space, since that would negate our purpose with having the Error line. We had some cases before we implemented the Error line policy where templates where silently ignoring significant lines because something in the data was different than we had seen before, and our regex didn't account for that. In order to prevent that from happening, we want to have something significant to match on for each line, or else raise an error.